src/onboarding/education/OnboardingEducationScreen.tsx

```diff
@@ -21,36 +20,27 @@ function useStep() {
   const { t } = useTranslation()
 
   return React.useMemo(() => {
-    const screenSet = Math.random() < 0.5 ? 0 : 1
-    const variant = screenSet === 0 ? 'old' : 'new'
-    const steps = shuffle([
+    // Order is highest performing from experiment - Spend, Impact, Payment
+    return [
       {
-        title: [t('onboardingEducation.step1'), t('onboardingEducation.payment')][screenSet],
+        title: t('onboardingEducation.step3'),
         isTopTitle: true,
-        image: onboardingEducation1,
+        image: onboardingEducation3,
         topic: EducationTopic.onboarding,
-        valueProposition: 'payment',
-        variant,
       },
       {
-        title: [t('onboardingEducation.step2'), t('onboardingEducation.impact')][screenSet],
+        title: t('onboardingEducation.step2'),
         isTopTitle: true,
         image: onboardingEducation2,
         topic: EducationTopic.onboarding,
-        valueProposition: 'impact',
-        variant,
       },
       {
-        title: [t('onboardingEducation.step3'), t('onboardingEducation.spend')][screenSet],
+        title: t('onboardingEducation.step1'),
         isTopTitle: true,
-        image: onboardingEducation3,
+        image: onboardingEducation1,
         topic: EducationTopic.onboarding,
-        valueProposition: 'spend',
-        variant,
       },
-    ]) as any
-    const order = steps.map((item: { valueProposition: string }) => item.valueProposition).join('-')
-    return { steps, variant, order }
+    ]
```

Comment:
nit: wondering if it may be confusing for localization, since stepX is no longer the real step X 😅

Reply:
I could see how that would be confusing, but as this page doesn't change often it should be ok.